modules/mono/glue/GodotSharp/GodotSharp/Core/Vector4I.cs

```diff
@@ -228,9 +228,23 @@ readonly get
         }
 
         // Constants
+        private static readonly Vector4I _min = new Vector4I(int.MinValue, int.MinValue, int.MinValue, int.MinValue);
+        private static readonly Vector4I _max = new Vector4I(int.MaxValue, int.MaxValue, int.MaxValue, int.MaxValue);
+
         private static readonly Vector4I _zero = new Vector4I(0, 0, 0, 0);
         private static readonly Vector4I _one = new Vector4I(1, 1, 1, 1);
 
+        /// <summary>
+        /// Min vector, a vector with all components equal to <see cref="Int32.MinValue"/>. Can be used as a negative integer equivalent of <see cref="Vector4.Inf"/>.
```

Comment:
nit: To be consistent with the rest of the documentation.

```suggestion
        /// Min vector, a vector with all components equal to <see cref="int.MinValue"/>. Can be used as a negative integer equivalent of <see cref="Vector4.Inf"/>.
```